docs-src/concepts/what-is-a-memo.md

```diff
@@ -8,4 +8,9 @@ tags:
   - explanation
 ---
 
-A Memo is a non-indexed user-supplied set of Workflow Execution metadata that is displayed with Filtered List results.
+A Memo is a non-indexed set of Workflow Execution metadata that users supply and that a Filtered List displays.
```

Comment:
what's a filtered list? a list of WEs that comes back from ListWorkflows with a List Filter? i'd link to definition

"user" sounds to me like end user, is that what you mean? do we use the term elsewhere in the docs? it could also be programmatically supplied by the dev